platform/pulpcore/app/models/content.py

```diff
@@ -79,24 +79,28 @@ class Content(MasterModel):
     """
     A piece of managed content.
 
-    Attributes:
-
-        natural_key_fields (tuple): Natural key fields.  Must be models.Field subclasses.
-
     Relations:
 
         notes (GenericKeyValueRelation): Arbitrary information stored with the content.
         artifacts (models.ManyToManyField): Artifacts related to Content through ContentArtifact
     """
     TYPE = 'content'
 
-    natural_key_fields = ()
-
     notes = GenericKeyValueRelation(Notes)
     artifacts = models.ManyToManyField(Artifact, through='ContentArtifact')
 
     class Meta:
         verbose_name_plural = 'content'
+        unique_together = ()
+
+    @classmethod
+    def natural_key_fields(cls):
+        """
+        Returns a list of the natural key fields which usually equates to unique_together fields
```

Comment:
This is a generator actually, so it yields field names as you iterate over the generator. If you want to convert it to return a list, I think that would be alright in this case. Either way, the docstring should match the behavior.